common/src/main/java/com/netflix/conductor/common/metadata/workflow/WorkflowDef.java

```diff
@@ -84,7 +85,7 @@
 	private boolean workflowStatusListenerEnabled = false;
 
 	@ProtoField(id = 11)
-	@NotEmpty(message = "ownerEmail cannot be empty")
+	@OwnerEmailMandatoryConstraint
```

Comment:
Hi, @skissane.

Nice PR. Thanks for working on it.

I would just rename this for ```OwnerEmailMandatory```. We already know that this is a Constraint because of the word ```mandatory```. What do you think?

Reply:
Well, if you look at the other annotations in `com.netflix.conductor.common.constraints` package, they've all got `Constraint` at the end of their name as well. So, even though I agree the word is a bit superfluous, it is consistent with the others.

Reply:
Makes sense, @skissane. Thank you.